### ISSUE TITLE
Cant disable "kinetic" scroll

### ISSUE THREAD

**Issue by hacxman** [xournalpp/xournalpp#4885]
**Affects versions :** 1.1.3
 - OS: Linux (Arch family, EndeavourOS)
 - (Linux only) Desktop environment: X11
 - libgtk: 3.24.37
 - Version of Xournal++: 1.1.3, 2dbd16d2
 - Installation method: distro, using pacman

**Describe the bug**
Xournal++ scrolls document after I've stopped scrolling,
slowing down the speed. Much like on mobile phone.
Problem is I wanted to scroll a bit (few lines maybe), not 3 or more pages.
It should never care how fast I've scrolled or how many lines.
Also I've noticed slight document jump in other direction.
I.e. scrolling down, but after a moment it slightly jumps up.
Was scrolling with touchpad and I am 100% sure that any "kinetic"(?)
scroll feature on touchpad is turned off (other programs scroll as usual).
Basically I never want this kinetic feature on, it prevents me from using
programs normally and comfortably as I am used to.

**To Reproduce**
1. Open document
2. Scroll
3. Observe

**Expected behavior**
Document should stop moving immediately after stopping scrolling.

**Additional context**
Making this at least configurable (on/off is enough for me) would dramatically increase usability.

Thank you very much!

**Comment by rolandlo** [comment]
Have you tried disabling kinetic scrolling in the Touchscreen tab of menu `Edit > Preferences`?

**Comment by christianhorne** [comment]
`Edit > Preferences > Touchscreen > Disable GTK's built-in touchscreen scrolling`
